release-calendar/src/client/api-service.ts

```diff
@@ -61,4 +61,10 @@ export class ApiService {
     );
     return new CurrentReleases(currentReleases);
   }
+
+  async getReleaseDates(requestedRelease: string): Promise<Release> {
```

Comment:
rename `getReleaseDates` because it's not getting `releaseDates` anymore.

Reply:
Oh, I just saw your comment. Okay.

Reply:
:)